### ISSUE TITLE
navigation via arrow keys 

### ISSUE THREAD

**Issue by elmirafar** [epha/kompendium#23]
would be awesome!
If search has enough letters, I wish to jump to the listed products and select it with enter.

**Comment by ephaee** [comment]
I agree, additionally we should support tab navigation, using tab key to focus the search result items.

Also selecting an existing product in the product menu should be supporting keyboard navigation
